### PR TITLE
pivpn: add page

### PR DESCRIPTION
**Pull request by sbrl**
Found this recently - it works like a charm! Just don't forget to allow it through your firewall like I did.....

### CODE REVIEW

File: pages/linux/pivpn.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# pivpn
+
+> Easy security-hardened OpenVPN setup and manager.
+> Originally designed for the Raspberry Pi, but works on other Linux devices too.
+> Homepage: <http://www.pivpn.io/>.
+
+- Install PiVPN:
```
Comment (agnivade) [inline pages/linux/pivpn.md]: We usually do not mention install steps. Let us remove this.

File: pages/linux/pivpn.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+- List all client devices:
+
+`sudo pivpn list`
+
+- List currently connected devices and their statistics:
+
+`sudo pivpn clients`
+
+- Revoke a previously authed device:
```
Comment (agnivade) [inline pages/linux/pivpn.md]: authenticated.

File: pages/linux/pivpn.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
+# pivpn
+
+> Easy security-hardened OpenVPN setup and manager.
+> Originally designed for the Raspberry Pi, but works on other Linux devices too.
```
Comment (agnivade) [inline pages/linux/pivpn.md]: I wonder if this line is necessary ?

File: pages/linux/pivpn.md (hunk 1 above)
Comment (sbrl) [reply pages/linux/pivpn.md]: Can do!

File: pages/linux/pivpn.md (hunk 3 above)
Comment (sbrl) [reply pages/linux/pivpn.md]: I think so. `Pi` is in the name, which is implies that it only work on Raspberry Pis.

File: pages/linux/pivpn.md (hunk 2 above)
Comment (sbrl) [reply pages/linux/pivpn.md]: Oops! Totally didn't see this comment here. It must have slipped through the net!

Changed.

**Pull request merged**